extensions/projection/README.md

```diff
@@ -0,0 +1,139 @@
+| proj:geometry    | Polygon Object  | Recommended. Defines the footprint of this item. |
+| proj:bbox        | [number]        | Recommended. Bounding box of the item in the native CRS |
+| proj:centroid    | Centroid Object | Recommended. Coordinates representing the centroid of the item in the native CRS |
+
+**proj:epsg** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data, and can usually be referenced using an [EPSG code](http://epsg.io).
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:epsg should be set to null. It should also be set to null if a CRS exists, but for which
+there is no valid EPSG code.
+
+**proj:crs** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data. This value is a PROJ string.
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:crs should be set to null. It should also be set to null if a CRS exists, but for which
+a PROJ string does not exist.
+
+**proj:geometry** - A Polygon object representing the footprint of this item, formatted according the Polygon object format specified in [RFC 7946, sections 3.1.6](https://tools.ietf.org/html/rfc7946), except not necessarily in EPSG:4326 as required by RFC7946.  Specified based on the `proj:crs` field (not necessarily EPSG:4326). Ideally, this will be represented by a Polygon with five coordinates, as the item in the native CRS should be a square aligned to the CRS grid.  It is recommended that either or both of `proj:geometry` and `proj:bbox` be defined.
+
+**proj:bbox** - Bounding box of the asset represented by this item in the native CRS. Specified as four coordinates based on the CRS defined in the `proj:epsg` and `proj:crs` fields.  First two numbers are coordinates of the lower left corner, followed by coordinates of upper right corner, e.g., \[west, south, east, north], \[xmin, ymin, xmax, ymax], \[left, down, right, up]. It is recommended that either or both of `proj:geometry` and `proj:bbox` be defined.
+
+**proj:centroid** - Coordinates representing the centroid of the item in the native coordinate system.  Coordinates are defined in latitude and longitude, even if the native coordinate system does not use lat/long.
+
+## Centroid Object
+
+This object represents the centroid of an asset's geometry.
+
+| Field Name          | Type   | Description                                                  |
+| ------------------- | ------ | ------------------------------------------------------------ |
+| lat                 | number | The latitude of the centroid.  |
+| lon                 | number | The longitude of the centroid. |
+
+## Examples
```

Comment:
Could we please make this a complete example and add it to the examples folder so that it can be used for validation etc?

Reply:
I'm wondering if centroid should live next to the top level bbox? Projections seems very specific, and centroid seems more like a quick lookup in a standardized way if you are specifying that is has to be lat long (WGS84 I assume as well?)